interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a hash in Ruby?
 
-  Your answer:
+  Your answer: I will have to do more research but I believe a hash is a key value pair.
 
-  Researched answer:
+  Researched answer: Unique keys that hold a specific value. Hashes are similar to an array but aren't limited to being indexed as an integer.
 
 
 
 2. What is a gem?
 
-  Your answer:
+  Your answer: Is the equivalent of a library file to yarn in Javascript
 
-  Researched answer:
+  Researched answer: A packaged ruby code that you can add to your application i.e rspec
 
 
 
 3. What is Ruby on Rails?
 
-  Your answer:
+  Your answer: an application that displays your code similarly to React and JS.
 
-  Researched answer:
+  Researched answer: A framework used to create websites and web based applications using Ruby
 
 
 
 4. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: database that has tables.
 
-  Researched answer:
+  Researched answer: A collection of data with values established relationships between them. These items are organized as a set of tables with columns and rows
 
 
 
 5. What are primary keys? Why are they important?
 
-  Your answer:
+  Your answer: Rows that are uniquely named to prevent conflation from locating and identifying them.
 
-  Researched answer:
+  Researched answer: Are unique IDs given to rows. They are important in identifying rows that may be conflated with similar row names or titles.  
```

Comment:
These are correct! And they would win you points in a game of trivia. I love trivia games! Interviews are more in depth than correct answers. Next week spend some time sharing how you've used the tools being discussed. And especially what you have found out about them from usage. Prove to me (your interviewer) you know what you are talking about.